src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BaselineHelper.cs

```diff
@@ -81,7 +81,7 @@ public static string DiffFiles(string file1Path, string file2Path, ITestOutputHe
         {
             (Process Process, string StdOut, string StdErr) diffResult =
                 ExecuteHelper.ExecuteProcess("git", $"diff --no-index {file1Path} {file2Path}", outputHelper);
-            Assert.Equal(1, diffResult.Process.ExitCode);
+            Assert.True(diffResult.Process.ExitCode == 0 || diffResult.Process.ExitCode == 1);
```

Comment:
Does it make sense to retain this Assert anymore?  It looks like the exit code from git diff is either 0 or 1.

Reply:
Yeah probably not. It looks like other error conditions return 1 as well so this doesn't add any value.